src/main/java/net/sf/jabref/model/entry/BibtexEntryType.java

```diff
@@ -35,23 +35,23 @@
 import net.sf.jabref.model.database.BibtexDatabase;
 
 /**
- * Provides a list of known entry types
- * <p>
- * The list of optional and required fields is derived from http://en.wikipedia.org/wiki/BibTeX#Entry_types
+ * Abstract base class for all entry types.
  */
 public abstract class BibtexEntryType implements Comparable<BibtexEntryType> {
```

Comment:
Idea: secondary optional only in biblatex, classes should reflect that. Boolean method if current class is biblatex class? Maybe interface + abstract classes for bibtex/biblatex?